src/main/groovy/org/shipkit/gradle/IncrementalReleaseNotes.java

```diff
@@ -57,6 +59,36 @@ public void setReleaseNotesFile(File releaseNotesFile) {
     }
 
     /**
+     * The version we are generating the release notes for.
+     */
+    @Input
+    public String getVersion() {
+        return version;
+    }
+
+    /**
+     * See {@link #getVersion()}
+     */
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    /**
+     * The tag prefix used for release note generation.
+     */
+    @Input
+    public String getTagPrefix() {
```

Comment:
Let's use Javadoc link to ```{@link ReleaseConfiguration.Git#getTagPrefix()}``` so that we don't have to duplicate the documentation.